### PR TITLE
Convert some Columns into Blocks

### PR DESCRIPTION
**Pull request by abarth**
These columns were secretly re-creating Block in a more complex way. Now we
just use Block directly.

### CODE REVIEW

**Comment by abarth** [comment]
@HansMuller @vlidholt

File: examples/material_gallery/lib/demo/tooltip_demo.dart [hunk 1]
```diff
@@ -19,8 +19,7 @@ class TooltipDemo extends StatelessWidget {
       ),
       body: new Builder(
         builder: (BuildContext context) {
-          return new Column(
-            crossAxisAlignment: CrossAxisAlignment.stretch,
```
Comment (eseidelGoogle) [inline examples/material_gallery/lib/demo/tooltip_demo.dart]: Does CrossAxisAlignment.stretch want a dartdoc to say that if you're using this you might want block, or am I reading too much into this?

Comment (abarth) [inline examples/material_gallery/lib/demo/tooltip_demo.dart]: Done.

**Comment by HansMuller** [comment]
LGTM

**Pull request merged**